src/Directory.Build.props

```diff
@@ -39,6 +39,9 @@
          https://github.com/Microsoft/msbuild/issues/2865-->
     <GenerateResourceMSBuildArchitecture>CurrentArchitecture</GenerateResourceMSBuildArchitecture>
     <GenerateResourceMSBuildRuntime>CurrentRuntime</GenerateResourceMSBuildRuntime>
+
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>$(NoWarn);1570;1572;1573;1587</NoWarn>
```

Comment:
What are those warnings? Shouldn't we leave them on?

Reply:
At least document what they are and if/when they should be removed from this list.

Reply:
These are the warnings about the XML doc comments.  There were hundreds of them.  I couldn't see that they were disabled before the toolset upgrade, but I checked a bit of the code in master, and it looked like the warnings would have been generated there too.  So I think somehow they were being ignored.